app/models/report.rb

```diff
@@ -1,37 +1,40 @@
 # frozen_string_literal: true
 
+require "csv"
+
 # Runs a query against solr and returns the results.
 # Does exactly what blacklight does, paginates the solr requests untill all results
 # have been received
-
 class Report
   include Blacklight::Configurable
-  include CsvConcern
   include DateFacetConfigurations
   include Blacklight::Searchable
 
   class << self
     include ValueHelper
   end
 
+  COLUMN_SELECTOR_COLUMN_SIZE = 3
+  ROWS_PER_PAGE = 100
+
   REPORT_FIELDS = [
     {
       field: :druid, label: "Druid",
       proc: ->(doc) { doc.druid },
-      sort: true, default: true, width: 100, download_default: true
+      sort: true, default: true, width: 100
     },
     {
       field: :purl, label: "Purl",
       proc: ->(doc) { File.join(Settings.purl_url, doc.druid) },
       solr_fields: %w[id],
-      sort: false, default: false, width: 100, download_default: true
+      sort: false, default: false, width: 100
```

Comment:
is it really the case that all field widths should be equal?  Or maybe the user can adjust this once it loads?

Reply:
All field widths are not equal, but most of them are set to `100`. These are effectively minimum widths which we can suss out via line 162.